keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandler.kt

```diff
@@ -873,6 +874,13 @@ class ClusterHandler(
     val healthy: Boolean = activeServerGroups.all {
       it.instanceCounts?.isHealthy(resource.spec.deployWith.health) == true
     }
+
```

Comment:
This event, emitted from the cluster handler and titus cluster handler, propagates the health information to the UnhealthyVeto